### ISSUE TITLE
Submodules in matRad 

### ISSUE THREAD

**Issue by weilidays** [e0404/matRad#424]
I am a new user of matRad, thanks all contributors for all your nice work, and for the very neat Wiki QuickSetup and Technical Documents. 

I followed the QuickSetup and did 'fork' and 'clone'. Errors showed up when I run example 4 and 6 of unitTest/, notifying me that related files were missing. I realized that my local 'MCsquare' and  'ompMC' directories were empty, and then figured out that they were git submodules. It may be  helpful to tell new users( who are not familiar with 'submodules') to use 'git clone --recurse-submodules' to download matRad in the QuickSetup document.... Just my two cents... Thanks.

**Comment by markbangert** [comment]
Brilliant idea. Included the info at https://github.com/e0404/matRad/wiki/How-to-get-a-local-copy.

**Comment by wahln** [comment]
Actually in the new version this shouldn't happen, since the binaries are incorporated. Did you use the new version or some alternative older branch?

**Comment by weilidays** [comment]
I forked the current master version. Here is $git show reply:
commit c22da7d2a593b7ae56277ce124074c55fe4d45dd (HEAD -> master, tag: 2.10.0, upstream/master, origin/master, origin/HEAD)
Merge: d63a2b0 8bf986c
Author: wahln <n.wahl@dkfz.de>
Date:   Fri Jun 5 16:54:35 2020 +0200

**Comment by wahln** [comment]
Okay, for me it doesnt require the submodules to be present. Probably there was no matching binary / pre-compiled binary for your system, which then required re-compilation of the files from source which are given in the submodules.
What system / what Matlab version are you on?

**Comment by weilidays** [comment]
Good point. I am using Mac OS 10.13. and Matlab 2020a.

**Comment by wahln** [comment]
Okay, I guess Mac OS is our least tested system, so some stuff might not work there out of the box. What confuses me a little is that the MCsquare and ompMC directories are empty after cloning. (There are ompMC and MCsquare folders directly in the matRad root which should contain the compiled files, and folders in the "submodules" folder which contain the source after cloning). But if it works I am happy.

**Comment by weilidays** [comment]
Oh man, I think you are right. I understand my problem better now. 

The error of missing files happened actually was not because the folders in the 'submodules' folder were empty. It because the MCsquare and ompMC under my local $matRadRoot were missing (and they were missing in my fork too). 

I noticed some directories were missing in my fork earlier,  so I deleted my fork and  forked again, make sure no directories were missing. Then I jumped to use 'git clone --recurse-submodule' ...

I just tested again, 'git clone' is enough, those complied files are there. No error of example 4 and 6. Sorry for any confusion. I am a little bit surprised that forked repository could be incomplete. Never experienced that before...
